config/language/es/views.view.group_nodes.yml

```diff
@@ -23,12 +23,15 @@ display:
           separator: ', '
         view_group_content:
           admin_label: 'View relation link'
+          label: 'Link to Group content'
```

Comment:
This isn't in Spanish, but nor is the text or admin label. SImilar for French.
Don't think it's an issue for this PR, but we could circle back on these translations.